lib/mycha_installer.coffee

```diff
@@ -0,0 +1,67 @@
+async = require 'async'
+{exec} = require 'child_process'
+fsExtra = require 'fs-extra'
+path = require 'path'
+
+
+class MychaInstaller
+
+  constructor: ({@cwd, @testHelperPath}) ->
+
+
+  install: (done) ->
+    async.parallel [
+      @_installDependencies
+      @_writeMychaConfig
+      @_writeTestHelper
+    ], done
+
+
+  _installDependencies: (done) =>
+    exec "npm install --save-dev chai sinon sinon-chai", {@cwd}, done
+
+
+  _writeMychaConfig: (done) =>
```

Comment:
What I want to see here is a more industrial scale solution for installing things into an existing repository. There are a number of edge cases that need to be properly handled by Mycha if we want it to be usable in serious projects. Like what happens if the test helper already exists. The user should have the option to
* keep his existing test helper
* replace the existing one with the new one
* have the installer merge the required changes into the existing test helper
* see a diff between the existing and the new helper to do the changes himself
* abort the install safely

I think the installer should also display which exact files and directories it has created, similar to how Rails generators work, so that the user knows what just happened.

There are (of course) libraries that provide such functionality. In Ruby I would use [Thor](https://github.com/erikhuda/thor). I haven't found a direct counterpart for that in Node. But there is [Yeoman](http://yeoman.io) (Grunt based) and [Slush](http://slushjs.github.io/#/) (Gulp based). 

Please check out Yeoman, since they have really nailed the UX part of this.
* npm install --global yeoman yo generator-node
* mkdir test ; cd test
* yo node
  * check out how comfortably it displays the options, default values, and allows to modify them
* run it again, and play with the different options (press "d" to show a diff if there are conflicts etc)

Thor also allows to [add content into existing files](https://github.com/kevgo/sections_rails/blob/master/lib/generators/sections_generator.rb#L9-L11). We might need that in case the spec helper already exists.

All of this can happen in a separate PR.